### PR TITLE
Blais5 3630

### PR DESCRIPTION
**Pull request by alistair-platt**

### CODE REVIEW

File: Blaise.Cati.Tests.Behaviour/app.config [hunk 1]
```diff
@@ -1,60 +1,61 @@
...
+		<section name="specFlow" type="TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow" />
+	</configSections>
+	<specFlow>
+		<stepAssemblies>
+			<stepAssembly assembly="Blaise.Tests.Behaviour" />
+			<stepAssembly assembly="Blaise.Cati.Tests.Behaviour" />
+		</stepAssemblies>
+	</specFlow>
+	<appSettings>
+		<add key="UninstallSurveyTimeOutInSeconds" value="5" />
```
Comment (elthorne) [inline Blaise.Cati.Tests.Behaviour/app.config:13]: Did you intend to commit a hard-coded variable?

Comment (alistair-platt) [reply Blaise.Cati.Tests.Behaviour/app.config:13]: just changing it

File: Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs [hunk 2]
```diff
@@ -15,42 +15,106 @@ public class InstrumentHelper
...
+        }
+
+        public void InstallInstrument(string instrumentPackage = "")
+        {
+            if (DoesSurveyExists(BlaiseConfigurationHelper.InstrumentName))
+            {
+                _blaiseSurveyApi.UninstallSurvey(BlaiseConfigurationHelper.InstrumentName,
+                    BlaiseConfigurationHelper.ServerParkName);
+                Thread.Sleep(int.Parse(BlaiseConfigurationHelper.UninstallSurveyTimeOutInSeconds) * 1000);
+            }
```
Comment (elthorne) [inline Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs:63]: Is this sleep for 5*1000 seconds (83 minutes) or have I misread this line?

File: Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs [hunk 3]
```diff
@@ -15,42 +15,106 @@ public class InstrumentHelper
...
+                CheckForErroneousSurvey(BlaiseConfigurationHelper.InstrumentName);
+            }
+            else
+            {
+                //The uninstall failed
+                throw new Exception($"Error trying to uninstall questionnaire {BlaiseConfigurationHelper.InstrumentName}");
+            }
         }
 
+        /*   public void InstallInstrument(string instrumentPackage)
```
Comment (elthorne) [inline Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs:81]: Is it worth removing commented code?

File: Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs (hunk 2 above)
Comment (alistair-platt) [reply Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs:63]: The * 1000 converts the seconds to milliseconds a requirement for the function

File: Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs (hunk 3 above)
Comment (alistair-platt) [reply Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs:81]: Just taking it out

**Review by tomphp — APPROVED** [review]

**Pull request merged**